examples/meta/src/clustering/gaussian_mixture_models.sg.in

```diff
@@ -6,7 +6,7 @@ Features features_train = create_features(f_feats_train)
 
 #![create_gmm_instance]
 int num_components = 3
-GMM gmm(num_components)
+GMM gmm(num_components, seed = 1)
```

Comment:
this will make the integration test fail if you don't update the data repo or?

Reply:
oh, this is a mistake, i should't change this, will change back.